integration_test/third_party_apps_test/applications/mariadb/metadata.yaml

```diff
@@ -364,17 +383,14 @@ configuration_options:
           default: null
           description: This value must be `mysql`.
         - name: endpoint
-          default: localhost:3306
-          description: The url exposed by MySQL.
+          default: /var/run/mysqld/mysqld.sock
+          description: The `hostname:port` or Unix socket path starting with `/` used to connect to the MySQL server.
```

Comment:
If the goal is to match the public docs, then should this be reverted? The docs say `localhost:3306` today: https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/mariadb#MariaDB-metrics

If this change was intentional, then should we change the nomenclature here from `MySQL server` to `MariaDB server`?

Reply:
You're right, sent a change to update the existing docs manually (since our current MySQL/MariaDB docs aren't generated). After that change is submitted and staged, this will match the docs.

> If this change was intentional, then should we change the nomenclature here from `MySQL server` to `MariaDB server`?

Good catch. Done.

Reply:
Fix CL submitted! Good catch, Jeff. Thanks.